src/comments.js

```diff
@@ -0,0 +1,146 @@
+/* eslint-disable no-await-in-loop */
+
+const baseURL = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps';
+
+const appID = '8a0dfW0tu0UAP5mKoeUq';
+
+export const fetchComments = async (showId) => {
+  const response = await fetch(`${baseURL}/${appID}/comments?item_id=${showId}`);
+  const data = response.ok ? await response.json() : [];
+  return data;
+};
+
+const saveComment = async (showId, commentData) => {
+  const data = {
+    item_id: showId,
+    username: commentData.name,
+    comment: commentData.comment,
+  };
+  const response = await fetch(`${baseURL}/${appID}/comments`, {
+    method: 'POST',
+    body: JSON.stringify(data),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+  return response.ok;
+};
+
+export const updateShowWithComments = async (shows) => {
+  let i = 0;
+  while (i < 24) {
+    const showComments = await fetchComments(shows[i].id);
+    shows[i].comments = showComments;
+    i += 1;
+  }
+  return shows;
+};
+
+export const countComments = (show) => show.comments.length;
+
+const displayComment = (comment) => {
+  const commentDiv = document.createElement('div');
+  commentDiv.classList.add('comment');
+
+  const commentHeaderDiv = document.createElement('div');
+  commentHeaderDiv.classList.add('comment-header');
+  const author = document.createElement('h4');
+  author.innerText = comment.username;
+  const timestamp = document.createElement('small');
```

Comment:
- [ ] Also make sure you follow the design. Currently, you are displaying all the comments below the form that doesn't look good and I've got confused as well that comments are not displaying. So, please follow the design and display the comment above the form.

### Your design

![Screenshot from 2021-12-24 17-39-45](https://user-images.githubusercontent.com/22434660/147354030-ef69b131-4e04-4fcc-9008-b7ea85017c1b.png)


### Given design
![Screenshot from 2021-12-24 18-22-28](https://user-images.githubusercontent.com/22434660/147354017-f6c445ee-2627-46e1-af5c-ce15dd1130e3.png)

Reply:
Ok done too. Thanks for reviewing again